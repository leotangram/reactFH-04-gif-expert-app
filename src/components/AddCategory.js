import React, { useState } from 'react'
import PropTypes from 'prop-types'

const AddCategory = props => {
  const [inputValue, setInputValue] = useState('Hola mundo')

  const handleInputChange = e => setInputValue(e.target.value)

  const handleSubmit = e => {
    e.preventDefault()
  }

  return (
    <form onSubmit={handleSubmit}>
      <h1>{inputValue}</h1>
      <input type="text" value={inputValue} onChange={handleInputChange} />
    </form>
  )
}

AddCategory.propTypes = {}

export default AddCategory
