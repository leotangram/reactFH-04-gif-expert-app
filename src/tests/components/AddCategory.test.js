import React from 'react'
import AddCategory from '../../components/AddCategory'
import { shallow } from 'enzyme'

describe('AddCategory', () => {
  test('should show correctly', () => {
    const setCategories = () => {}
    const wrapper = shallow(<AddCategory setCategories={setCategories} />)
    expect(wrapper).toMatchSnapshot()
  })
})
