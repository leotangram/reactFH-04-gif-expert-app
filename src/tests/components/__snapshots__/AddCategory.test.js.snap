// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddCategory should show correctly 1`] = `
<form
  onSubmit={[Function]}
>
  <h1 />
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
